doc/backend_vars.asciidoc

```diff
@@ -6,7 +6,7 @@ Supported variables per backend
 [options="header",cols="^m,^m,^m,v",separator=";"]
 |====================
 Variable;Values allowed;Default value;Explanation
-CASEDIR;string;;Path to test distribution. Can be a git repository URL of a test distribution to checkout with an optional refspec into the current directory. Tries to follow the definition of https://docs.npmjs.com/files/package.json#git-urls-as-dependencies , for example `git@github.com:os-autoinst/os-autoinst-distri-opensuse.git#feature/test`
+CASEDIR;string;;Path to test distribution. Can be a git repository URL of a test distribution to checkout with an optional refspec into the current directory. It tries to follow the definition of https://docs.npmjs.com/files/package.json#git-urls-as-dependencies (e.g. `https://github.com/os-autoinst/os-autoinst-distri-opensuse.git#feature/test` or `git@github.com:os-autoinst/os-autoinst-distri-opensuse.git`). Note that cloning via SSH is not possible if git is not configured accordingly on the worker host.
```

Comment:
I suggest to avoid the double negation so instead of `is not possible if git is not configured accordingly` write `is only possible if git is configured accordingly`. Also "worker host" is openQA speak which should be avoided in os-autoinst content. Just end the sentence after "accordingly."

Reply:
> Also "worker host" is openQA speak which should be avoided in os-autoinst content.

But openQA nevertheless exists and different hosts might be involved. To make it clear that neither the openQA web UI host or the local host are relevant I changed it to "on the host where isotovideo is executed".